src/statement.rs

```diff
@@ -55,6 +55,21 @@ impl<'l> Statement<'l> {
         unsafe { ffi::sqlite3_column_count(self.raw.0) as usize }
     }
 
+    /// Return the name of a column in the statement
+    #[inline]
+    pub fn column_name(&self, i: usize) -> &str {
+        unsafe {
```

Comment:
I think it would be helpful to add a `debug_assert!` here. Have a look at how it is throughout `statement.rs`. What do you think?

Reply:
ok